frontend/actions/updateUser.js

```diff
@@ -0,0 +1,14 @@
+import PipelineRequest from '@shopgate/pwa-core/classes/PipelineRequest';
+import { PIPELINE_UPDATE_USER } from './../constants/Pipelines';
+import { updateUser, updateUserSuccess, updateUserFailed } from './../action-creators';
+
+export default ({ mail: ignore, password: ignore2, ...restUser }) => (dispatch) => {
```

Comment:
Not sure if this is the right place to kick out these.  A function should probably not expect to get parameters that won't be used. For changing mail and password we will have other actions in the future anyway.